videos.js

```diff
@@ -0,0 +1,32 @@
+const videos = {
```

Comment:
What do you think about storing the videos in an array?

Reply:
Basically, having a hash with key at index values acts the same as an array that inherently has these index values. Resolved: fe787d32a319dd0eb330af376ce30ec82ed307d7